projects/igniteui-angular/src/lib/grids/grid/grid-filtering-ui.spec.ts

```diff
@@ -3322,6 +3322,37 @@ describe('IgxGrid - Filtering actions - Excel style filtering #grid', () => {
             checkboxes.forEach(c => expect(c.checked).toBeFalsy());
         }));
 
+        it('Should show the previously entered filter value when reopen esf dialog.', fakeAsync(() => {
```

Comment:
I think this test verifies the single case when clicking the Custom filter, does it. Since opening the Custom filter works differently than all other conditions (Custom loads the dialog with the current conditions, while clicking any other condition resets the expressions (unless clicking on the condition that is already applied). Let's have the test name be more descriptive about it
```suggestion
        it('Should show the previously entered filter value when reopen esf dialog from Custom filter menu.', fakeAsync(() => {
```

Reply:
Let's use the same bed to verify that the excel filter icon has the filtered class applied:

            const activeFilterIcon = header.nativeElement.querySelector('.igx-excel-filter__icon--filtered');
            expect(activeFilterIcon).toBeDefined('no active filter icon was found');
            
            
![image](https://github.com/IgniteUI/igniteui-angular/assets/14248932/d75c7f8a-fe5f-450e-8e13-3e13f2bd2b72)